zoe-core/src/functions/poll.kt

```diff
@@ -52,18 +53,28 @@ val poll = zoeFunction<PollConfig, PollResponse>(name = "poll") { config ->
 
         val filtered =
             records
-                .map {
+                .map { record ->
                     PolledRecord(
-                        key = it.key()?.toString(),
-                        offset = it.offset(),
-                        timestamp = it.timestamp(),
-                        partition = it.partition(),
-                        topic = it.topic(),
-                        formatted = it.value()?.let { rec -> jsonifier.format(rec) } ?: NullNode.getInstance()
+                        meta = RecordMetadata(
+                            key = record.key()?.toString(),
```

Comment:
Does key support avro schema? I'm wondering what would be output in case of avro keys.

Reply:
Currently, if the key is an avro record, the `toString()` here will transform it into a valid json record. So basically, it will appear in the output of zoe as a json string. Makes sense?